bundle/manifests/opentelemetry-operator.clusterserviceversion.yaml

```diff
@@ -65,13 +65,13 @@ metadata:
     categories: Logging & Tracing,Monitoring
     certified: "false"
     containerImage: ghcr.io/open-telemetry/opentelemetry-operator/opentelemetry-operator
-    createdAt: "2023-11-17T13:24:32Z"
+    createdAt: "2023-12-04T20:02:17Z"
     description: Provides the OpenTelemetry components, including the Collector
     operators.operatorframework.io/builder: operator-sdk-v1.29.0
     operators.operatorframework.io/project_layout: go.kubebuilder.io/v3
     repository: github.com/open-telemetry/opentelemetry-operator
     support: OpenTelemetry Community
-  name: opentelemetry-operator.v0.89.0
+  name: opentelemetry-operator.v0.89.0-30-g8a916dd
```

Comment:
you'll want to remove the suffix here (i.e. leave `opentelemetry-operator.v0.89.0`)